src/domain/entities/Questionnaire/QuestionnaireQuestion.ts

```diff
@@ -152,100 +152,102 @@ export class QuestionnaireQuestion {
     }
 
     static updateQuestions(
```

Comment:
Only if we have time. This are the kind of logic that really benefits from having some tests, as it's not trivial and extremely easy to introduce a bug. No need to cover *all* cases, but a subset that are really used in the app and appear tricky.

Reply:
Noted, will do this if i have time remaining after creation of ppt for meeting.